src/sfeQwiicUltrasonic.cpp

```diff
@@ -29,13 +40,33 @@ sfeTkError_t sfeQwiicUltrasonic::isConnected()
     return _theBus->ping();
 }
 
-sfeTkError_t sfeQwiicUltrasonic::triggerAndRead(uint16_t &distance)
+sfeTkError_t sfeQwiicUltrasonic::getDistance(uint16_t &distance)
+{
+    size_t bytesRead = 0;
+    uint8_t rawData[2] = {0, 0};
+    sfeTkError_t err;
+
+    _theBus->writeByte(kUltrasonicDistanceReadCommand);
+    err = _theBus->readRegisterRegion(_theBus->address(), rawData, 2, bytesRead);
+
+    // Check whether the read was successful
+    if (err != kSTkErrOk)
+        return err;
+
+    // Store raw data
+    distance = (rawData[0] << 8) | rawData[1];
+
+    // Done!
+    return kSTkErrOk;
+}
```

Comment:
The only difference I see between this and getTriggeredDistance() is sending the read command. Functionally, how are these different? Do we need both?

Reply:
I removed the `getTriggeredDistance` function, you're either going full I2C or not.